cartridge/remote-control.lua

```diff
@@ -421,10 +482,15 @@ local function drop_connections()
     table.clear(vars.handlers)
 
     for s, handler in pairs(handlers) do
-        if handler ~= fiber.self() then
-            pcall(function() handler:cancel() end)
-            pcall(function() s:close() end)
+        if s ~= fiber.self().storage.socket
```

Comment:
When is it possible?

Reply:
```
handler1    handler2
   |           |______________
   |           |              |
fiber1       fiber2         fiber3
   |           |              |
 task1       task2          task3
               |
        drop_connection   
```
In such situation `fiber2` will drop all handlers and tasks started by them except those with which he shares socket.